### ISSUE TITLE
tmt needs to recognize multiple IP addresses for multihost 

### ISSUE THREAD

**Issue by thrix** [teemtee/tmt#2620]
In Testing Farm public ranch due to the AWS setup we need to get multihost testing working:

* the multihost machines need to communicate using private IP addreses
* tmt will connect to the machines using public IPs

So we need two sets of IP addresses for each guest, correctly setup so they are used as described above.

**Comment by psss** [comment]
This is an important blocker for the Satellite team, raising priority.

**Comment by thrix** [comment]
We decided with @happz that we will implement some support needed for this in Artemis first, and then do other implementation of this issue to use that feature of Artemis to achieve what we need. While my solution would be quicker, it would not be feature proof and this way it will be better once we would need multihost communication between two clouds in the future.

Tentatively proposing for 1.58 and working on the Artemis change in this sprint

**Comment by psss** [comment]
Changes blocking this should have landed yesterday. @thrix, is that the case or are we still blocked here?

**Comment by thrix** [comment]
@psss yeah, no, Artemis changes are not yet even written. Anyway, I believe I can work on this without the support, it could just start working once Artemis will support it. And of course I will try to push out the Artemis changes finally this week, sorry for the delay.
